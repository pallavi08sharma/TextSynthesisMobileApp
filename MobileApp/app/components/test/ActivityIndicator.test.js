import React from 'react';
import renderer from 'react-test-renderer';

import ActivityIndicator from '../ActivityIndicator'

describe('<ActivityIndicator />', () => {
    it('renders correctly', async() => {
      const tree = renderer.create(<ActivityIndicator />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });