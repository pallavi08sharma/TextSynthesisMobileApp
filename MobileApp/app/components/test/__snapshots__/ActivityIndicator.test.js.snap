// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActivityIndicator /> renders correctly 1`] = `null`;
